resources/ory/profile-evaluation.yaml

```diff
@@ -3,26 +3,34 @@ global:
   ory:
     hydra:
       persistence:
-        enabled: false
+        enabled: true
```

Comment:
I'm not sure what to do with these ory changes. These are related to the fact that the azure_lite runtime plan uses the evaluation profile but it needs persisted ory-hydra unlike trial/free runtimes. 

@kyma-project/goat 
Should these changes be made here in the evaluation profile, or should we just keep them as runtime overrides for azure_lite until ory is fully deprecated and removed from runtimes? AFAIK ory is not getting modularized as being a deprecated component, right?

Also, imo changing the persistence is not allowed for existing ory deployments as it would cause statefuset to be changed in immutable fields. So this changeset on ory/profile-evaluation.yaml might not be even valid.

Reply:
@ebensom 
Yes, Hydra is deprecated and will be removed in the future. We agree that it would be better to just keep the overrides for azure_lite in place.

Reply:
@triffer okay. I've disabled ory hydra persistence